frontend/src/components/DropGem/AttachMedia.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <h1>Attach Media</h1>
+</template>
```

Comment:
Could you add a newline to the end of the file?

Reply:
Done